website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Private Networks
+description: Running ClusterLink in a private network when the K8s cluster is behind a NAT or firewall.
+---
+
+
+This task involves connecting ClusterLink behind a NAT or firewall.
+To connect the ClusterLink gateway, each peer should have a public IP that will be reachable from other peers to enable cross-cluster communications. In many cases, this is not possible because clusters are behind corporate NAT or firewalls that allow outgoing connections only. For this scenario, we will use the [Fast Reverse Proxy][] (FRP) open-source project to create reverse tunnels and connect all clusters behind a private network. With FRP, only one IP needs to be public to connect all the clusters in the fabric.
+
+To create connectivity between the ClusterLink gateways, we need to set up one FRP server with a public IP and create an FRP client for each ClusterLink gateway, as illustrated below.
```

Comment:
enable instead of create?
Also, the FRP client is needed only for gateways that are behind NAT, right?
The way it is worded seems to suggest that we create a FRP server as the hub of all communications and all gateways should use an FRP client.
I think it should be clearer that (1) needed only for gateways behind NAT/FW; (2) different FRP servers can be used (e.g., in close proximity to the cluster); and (3) the setup below is a only an example, not necessarily the only/recommended way, and it uses a single FRP server running inside a k8s cluster for two different gateways.